### PR TITLE
Output function params

### PR DESCRIPTION
**Pull request by MarshallOfSound**
Parse the function parameters that have been specified in electron/electron/pull/7596

/cc @zeke

### CODE REVIEW

**Comment by zeke** [comment]
Looks good. Can you add a test that asserts that these parameters are present on some of the functions?

**Comment by zeke** [comment]
The `webRequest.onBeforeRequest` callback might be a good one to test because it's slightly more complex:

screen shot 2016-10-13 at 9 41 24 am

**Comment by MarshallOfSound** [comment]
Test added 👍

File: test/index.js [hunk 1]
```diff
@@ -263,6 +263,19 @@ describe('APIs', function () {
       })
       expect(assertions).to.be.above(80)
     })
+
+    it('should have function parameters that than have deep documented parameters', function () {
```
Comment (zeke) [inline test/index.js]: typo: that than

File: test/index.js [hunk 2]
```diff
@@ -263,6 +263,19 @@ describe('APIs', function () {
...
+      var method = apis.WebRequest.instanceMethods.onBeforeRequest
+      var fnParam = method.parameters[1]
```
Comment (MarshallOfSound) [inline test/index.js]: Sorted 👍

**Comment by MarshallOfSound** [comment]
@zeke PR rebased and tests should be passing 👍

**Comment by zeke** [comment]
🍰 

1.10.0

**Pull request merged**